### PR TITLE
PT Translation Fixes

### PR DESCRIPTION
**Pull request by Kitsune912**

### CODE REVIEW

File: src/data/country.csv [hunk 1]
```diff
@@ -294,7 +294,7 @@ Sumatra,Sumatra,Sumatra,Sumatra,Sumatra,Sumatra,Суматра,Sumatra,Sumatra,S
 Sicily,Sizilien,Sicilia,Sicile,Sicilia,Sicílie,Сицилия,Sicilië,Sicilien,Sicília,西西里岛(Sicily)
 Mayotte,Mayotte,Mayotte,Mayotte,Mayotte,Mayotte,Майотта,Mayotte,Mayotte,Maiote,马约特岛(Mayotte)
 Réunion,La Réunion,Reunión,La Réunion,Réunion,Réunion,Реюньон,Réunion,Réunion,Reunião,留尼汪岛(Réunion)
-Falkland Islands,Falklandinseln,Islas Malvinas,Îles Malouines,Falklandsøyene,Falklandy,Фолклендские острова,Falklandeilanden,Falklandsöarna,Ilhas Malvinas,福克兰群岛(Falkland Islands)
+Falkland slands,Falklandinseln,Islas Malvinas,Îles Malouines,Falklandsøyene,Falklandy,Фолклендские острова,Falklandeilanden,Falklandsöarna,Ilhas Malvinas,福克兰群岛(Falkland Islands)
```
Comment (aplaice) [inline src/data/country.csv:297]: I think that this is a typo. (It should stay Falkland Islands.)

**Comment by aplaice** [comment]
Thanks very much! :)

Also, thanks for noticing that the capitals of Puerto Rico and Antigua and Barbuda are the same in Portuguese and hence we need a capital hint! :)

(Build is probably failing due to the Falkland Islands change.)

**Review by aplaice — APPROVED** [review]

**Comment by Kitsune912** [comment]
Everything fixed ;)

**Review by axelboc — APPROVED** [review]
Thanks!

**Comment by aplaice** [comment]
Squashing (I hope that's OK!) and merging!

Thanks again!

**Pull request merged**